### PR TITLE
Release `1.1.2`

### PR DESCRIPTION
**Pull request by dmitrykuzmin**
This PR does the release of a library version `1.1.2`.

It also migrates all tests off Mockito and removes Mockito and SLF4J from the project dependencies.

### CODE REVIEW

**Comment by dmitrykuzmin** [comment]
@armiol PTAL.

File: datastore/src/test/java/io/spine/server/storage/datastore/tenant/NamespaceIndexTest.java [hunk 1]
```diff
@@ -110,20 +107,19 @@ void testNulls() {
...
         IterableSubject assertIds = assertThat(ids);
         assertIds.isNotNull();
-        assertIds.hasSize(1);
+        assertIds.hasSize(idCount + 1);
 
-        TenantId actual = ids.iterator()
-                             .next();
-        assertEquals(newId, actual);
+        assertThat(ids)
+                .contains(newId);
```
Comment (armiol) [inline datastore/src/test/java/io/spine/server/storage/datastore/tenant/NamespaceIndexTest.java]: Should we move this back a line?

File: datastore/src/test/java/io/spine/server/storage/datastore/type/DsColumnTypesTest.java [hunk 2]
```diff
@@ -118,7 +125,10 @@ void testLong() {
 
         setSimpleType(type, value);
 
-        verify(entity).set(eq(RANDOM_COLUMN_LABEL), eq(LongValue.of(value)));
+        BaseEntity<Key> entity = entityBuilder.build();
+        long entityField = entity.getLong(COLUMN_LABEL);
+        assertThat(entityField)
+                .isEqualTo(value);
```
Comment (armiol) [inline datastore/src/test/java/io/spine/server/storage/datastore/type/DsColumnTypesTest.java]: Should we move this back a line? Similar cases below.

**Review by armiol — APPROVED** [review]
@dmitrykuzmin LGTM with some comments to consider.

**Pull request merged**